wolfssl/openssl/evp.h

```diff
@@ -1102,6 +1102,7 @@ WOLFSSL_API int wolfSSL_EVP_DigestVerifyInit(WOLFSSL_EVP_MD_CTX *ctx,
 WOLFSSL_API int wolfSSL_EVP_Digest(const unsigned char* in, int inSz, unsigned char* out,
                               unsigned int* outSz, const WOLFSSL_EVP_MD* evp,
                               WOLFSSL_ENGINE* eng);
+WOLFSSL_API const char* wolfSSL_EVP_CIPHER_type_string(unsigned int type);
```

Comment:
Is this supposed to be a compatibility layer function? Typically we also add a macro to expose it.